packages/libs/error-stack/src/fmt/color.rs

```diff
@@ -68,12 +66,7 @@ static COLOR_OVERRIDE: AtomicOverride<ColorMode> = AtomicOverride::new();
 impl Report<()> {
     /// Set the color mode preference
     ///
-    /// If the value is [`None`], a previously set preference will be unset, while with [`Some`] a
-    /// specific color mode will be set, this mode will be used, otherwise if `detect` is enabled
-    /// the capabilities of the terminal are queried through [`owo_colors`].
-    ///
-    /// The value defaults to [`ColorMode::Emphasis`] (if the terminal supports it), otherwise
-    /// [`ColorMode::None`].
+    /// The value defaults to [`ColorMode::Emphasis`], otherwise [`ColorMode::None`].
```

Comment:
What do you mean by "otherwise `ColorMode::None`"?

Reply:
whoops, that must've been left there when we had the different color modes depending on the feature in #1800, removed in [1a19702](https://github.com/hashintel/hash/pull/1980/commits/1a1970264f0551a255344db50ec11613e5f2aed0)